src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -1014,18 +1039,18 @@ bool ReconstructionEngine_sequentialSfM::getBestInitialImagePairs(std::vector<Pa
   
   const unsigned iMin_inliers_count = 100;
   // Use a min angle limit to ensure quality of the geometric evaluation.
-  const float fRequired_min_angle = 5.0f;
+  const float fRequired_min_angle = _minAngleInitialPair;
   // Use a max angle limit to ensure good matching quality.
-  const float fLimit_max_angle = 40.0f;
+  const float fLimit_max_angle = _maxAngleInitialPair;
```

Comment:
same here, _maxAngleInitialPair is declared as double, maybe float is more than enough